.github/workflows/lint.yml

```diff
@@ -14,7 +14,7 @@ jobs:
         uses: actions/checkout@v3
       - uses: actions/setup-go@v3
         with:
-          go-version: 1.19.x
+          go-version: "1.19"
```

Comment:
The lint GitHub action is likely not ready for 1.20, which is fine. We'll wait until it's supported.